orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/strategies/MonitoredDeployStrategy.groovy

```diff
@@ -385,6 +396,28 @@ class MonitoredDeployStrategy implements Strategy {
       SyntheticStageOwner.STAGE_AFTER
     )
 
+    MonitoredDeployStageData stageData = parent.mapTo(MonitoredDeployStageData.class)
+    if (stageData.deploymentMonitor.id) {
+      def evalContext = stageData.getChildStageContext()
+      MonitoredDeployInternalStageData internalStageData = new MonitoredDeployInternalStageData()
```

Comment:
is there anything there to indicate that the deployment failed?

Reply:
not yet... there will be! for now, I just want to unblock a use case
https://github.com/spinnaker/orca/blob/master/orca-deploymentmonitor/src/main/java/com/netflix/spinnaker/orca/deploymentmonitor/models/DeploymentCompletedRequest.java#L35